### ISSUE TITLE
First line of input CSV should contain the data type

### ISSUE THREAD

**Issue by GalateaRaj** [galatea-associates/kafka-poc#62]
This data type string should match the data types listed in our google doc and determine what is used as key as well as which topic the messages are published to.

**Comment by gala-emile** [comment]
This can easily be done. I am not quite sure however I understand the purpose of it - would we want to do this so that just from the CSVs, we can see the data type we are dealing with and its respective keys?

**Comment by GalateaRaj** [comment]
Yes.  To improve readability and make it clear from the file what the data type is.
